### DISCUSSION TITLE
Incremental Learning of Object Detection Example

### DISCUSSION THREAD

**Discussion by onepiece010938** [ContinualAI/avalanche#764] [Feature Request]
Hi~ I'm new in continual learning.
I am extremely happy to find this open source library. I am currently assigned to study incremental learning of object detection. I want to use Avalanche to develop, but I haven't seen any examples of object detection in the Tutorial. I hope someone can provide me with some examples for reference or tell me how to get started.

**Comment by vlomonaco** [comment]
I think the best person to answer your question would be @lrzpellegrini :) 

Unfortunately we don't yet have ready-to-use examples, but once you get familiar with the library in general you should be able to easily integrate your own detection datasets and algorithms. You can ping us here or [on slack](https://join.slack.com/t/continualai/shared_invite/enQtNjQxNDYwMzkxNzk0LTBhYjg2MjM0YTM2OWRkNDYzOGE0ZTIzNDQ0ZGMzNDE3ZGUxNTZmNmM1YzJiYzgwMTkyZDQxYTlkMTI3NzZkNjU) in case you get stuck somewhere. 

You can aslo open a PR with your minimal example so that we can give you feedback on how to improve it! :)
